src/app/collective-rewards/shared/components/Table/TableHeaders.tsx

```diff
@@ -14,27 +14,32 @@ export type ISortConfig = {
 export type TableHeader = {
   label: string
   className: string
+  hiddenClassName?: string
   tooltip?: TooltipProps
   sortKey?: string
+  canBeHidden?: boolean
 }
 
 type TableHeaderProps = {
   tableHeader: TableHeader
   onSort?: (parameter: any) => void
   sortConfig?: ISortConfig
+  isHidden?: boolean
 }
 
 export const TableHeaderCell: FC<TableHeaderProps> = ({
-  tableHeader: { label, className, sortKey, tooltip },
+  tableHeader: { label, className, hiddenClassName, sortKey, tooltip, canBeHidden },
   onSort,
   sortConfig,
+  isHidden,
 }) => {
   return (
     <TableCell
       key={label}
       className={cn(
+        isHidden ? hiddenClassName : className,
```

Comment:
Should we think about hiddenClassName as an overwrite of className? Otherwise we may need to repeat the same properties in both.

Reply:
I don't get this comment.